EIPS/eip-1967.md

```diff
@@ -4,7 +4,8 @@ title: Standard Proxy Storage Slots
 description: Standardise where proxies store the address of the logic contract they delegate to, as well as other proxy-specific information.
 author: Santiago Palladino (@spalladino)
 discussions-to: https://ethereum-magicians.org/t/eip-1967-standard-proxy-storage-slots/3185
-status: Review
+status: Last Call
+last-call-deadline: 2022-01-16
```

Comment:
I know that we editors have been holding up the process quite a bit, but this'll need to be 14 days from when this pull request is merged :sweat_smile: